test-runner/src/main/scala/com/typesafe/spark/test/mesos/framework/runners/MultiClusterModeRunner.scala

```diff
@@ -0,0 +1,129 @@
+package com.typesafe.spark.test.mesos.framework.runners
+
+import java.io.File
+import java.net.URL
+
+import Utils._
+import com.typesafe.config.{Config, ConfigFactory}
+import com.typesafe.spark.test.mesos.mesosstate.MesosCluster
+
+import org.apache.zookeeper.{ZKUtil, ZooKeeper, KeeperException}
+
+import scala.sys.process.Process
+
+object MultiClusterModeRunner {
```

Comment:
The main thing I don't like about this is that `mit` no longer runs the entire test suite.  We have to run both `mit` and `mcm` to run all the tests, even though these two sets of tests are runnable on the same cluster.  These test don't seem to necessitate a separate test runner any more than any of the other tests we have.